experimental/CHANGELOG.md

```diff
@@ -7,6 +7,8 @@ All notable changes to experimental packages in this project will be documented
 
 ### :boom: Breaking Change
 
+* feat(exporter-prometheus)!, feat(shim-opencensus)!: drop support for the deprecated `type` field on `MetricDescriptor` [#5291](https://github.com/open-telemetry/opentelemetry-js/pull/5291)
```

Comment:
Not exactly sure what to say here, there is probably more to say, since, [as the test shown](experimental/packages/opentelemetry-exporter-prometheus/test/PrometheusExporter.test.ts) this change can have noticeable impact